stripe/src/main/java/com/stripe/android/view/StripeEditText.java

```diff
@@ -26,7 +27,7 @@
  * but we listen here for hardware key presses, older Android soft keyboard delete presses,
  * and modern Google Keyboard delete key presses.
  */
-public class StripeEditText extends AppCompatEditText {
+public class StripeEditText extends TextInputEditText {
```

Comment:
`TextInputEditText` extends `AppCompatEditText` -- just gives the containing `TextInputLayout` more control over it